src/rules/consistent-test-it.ts

```diff
@@ -26,7 +30,14 @@ export default {
         additionalProperties: false,
       },
     ],
+    type: 'suggestion',
   },
+  defaultOptions: [
+    { fn: 'test', withinDescribe: 'it' } as {
+      fn?: 'it' | 'test';
```

Comment:
do we need `as const` on the schema?

Reply:
might not need the type info here then

Reply:
Not as far as I'm aware - the `schema` is json schema, so it's not actually used by TS (it'd be cool if it was, but it's far too complex)